extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -107,8 +122,13 @@ class AmpStickyAd extends AMP.BaseElement {
     if (scrollTop > viewportHeight) {
       this.removeOnScrollListener_();
       this.deferMutate(() => {
+        this.visible_ = true;
+        setStyles(this.element, {
```

Comment:
will force visibility using `!important`, will change the style by adding classList now.